src/pages/SendCredentials/SendCredentials.js

```diff
@@ -136,14 +136,14 @@ const Verifiers = () => {
 						style={{ maxHeight: '80vh' }}
 					>
 						{filteredVerifiers.map((verifier) => (
-							<li
+							<button
```

Comment:
These `<button>`s are now children of a `<ul>`, perhaps that should be changed to `<div>` or something like that instead?